azkaban-common/src/test/bash/test_util.shunit2

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env shunit2
+# Make sure to "brew install shunit2" or use your favorite package manager to install
+# before running tests
+
+function oneTimeSetUp() {
```

Comment:
oneTimeSetup() is a shunit2 feature that is intended to prepare a environment for testing.  If temp directories are needed for testing, oneTimeSetup is when  they get created.   It seemed an appropriate place for sourcing the utils.sh file.  

FWIW: there's also oneTimeTeardown() useful for cleanup after tests are completed.  More info can be found here: https://github.com/kward/shunit2